website/content/docs/operators/deployment/schedulers/yarn.md

```diff
@@ -0,0 +1,123 @@
+containers are allocated for each topology.
+
+### Configuring the Heron client classpath
+
+1. Command `hadoop classpath` provides a list of jars needed to submit a hadoop job. Copy all jars
+to `HERON_INSTALL_DIR/lib/scheduler`.
+  * Do not copy commons-cli jar if it is older than version 1.3.1.
+1. Create a jar containing core-site.xml and yarn-site.xml. Add this jar to
+`HERON_INSTALL_DIR/lib/scheduler` too.
+
+> The YARN scheduler needs Hadoop and YARN jars in the classpath. Heron currently does not allow
+> customizing client classpath. Hadoop and YARN jars are version dependent and should not be
+> bundled with the Heron distribution. So, this manual step is needed till this Heron issue
+> ([271] (https://github.com/twitter/heron/issues/271)) is fixed.
+
+### Configure the YARN scheduler
+
+A set of default configuration files are provided with Heron in the [conf/yarn]
+(https://github.com/twitter/heron/tree/master/heron/config/src/yaml/conf/yarn) directory.
+The default configuration uses the local state manager. This will work with single-node local
+YARN installation only. A Zookeeper based state management will be needed for topology
+deployment on a multi-node YARN cluster.
+
+1. Custom Heron Launcher for YARN: `YarnLauncher`
+1. Custom Heron Scheduler for YARN: `YarnScheduler`
+1. State manager for multi-node deployment:
+`com.twitter.heron.statemgr.zookeeper.curator.CuratorStateManager`
+1. `YarnLauncher` performs the job of uploader also. So `NullUploader` is used.
+
+## Topology management
+
+### Topology Submission
+**Command**
+
+`$ heron submit yarn heron-examples.jar com.twitter.heron.examples.AckingTopology AckingTopology`
+
+**Sample Output**
+
+```
+INFO: Launching topology 'AckingTopology'
+...
+...
+Powered by
+     ___________  ______  ______  _______
+    /  ______  / /  ___/ /  ___/ /  ____/
+   /     _____/ /  /__  /  /__  /  /___
+  /  /\  \     /  ___/ /  ___/ /  ____/
+ /  /  \  \   /  /__  /  /__  /  /
+/__/    \__\ /_____/ /_____/ /__/
+
+...
+...
+com.twitter.heron.scheduler.yarn.ReefClientSideHandlers INFO:  Topology AckingTopology is running, jobId AckingTopology.
+```
+
+**Verification**
+
+Visit the YARN http console or execute command `yarn application -list` on a yarn client host.
+
+```
+Total number of applications (application-types: [] and states: [SUBMITTED, ACCEPTED, RUNNING]):1
+                Application-Id	    Application-Name	    Application-Type	      User	     Queue	             State	       Final-State	       Progress	                       Tracking-URL
+application_1466548964728_0004	      AckingTopology	                YARN	     heron	   default	           RUNNING	         UNDEFINED	             0%	                                N/A
+```
+
+### Topology termination
+**Command**
+
+`$ heron kill yarn AckingTopology`
+
+
+###Log File location###
+Assuming HDFS as the file system, Heron logs and REEF logs can be found in the following locations
```

Comment:
Colon after this sentence, and also add an empty line between this and the ordered list items